docs/design/autoware-interfaces/ad-api/types/autoware_adapi_v1_msgs/msg/MrmState.md

```diff
@@ -18,7 +18,7 @@ uint16 MRM_OPERATING = 2
 uint16 MRM_SUCCEEDED = 3
 uint16 MRM_FAILED = 4
 
-# For behavior
+# For behavior. Deprecated: use description API.
```

Comment:
Maybe it's better if we insert the link to description API page.

Reply:
Add URL https://github.com/autowarefoundation/autoware-documentation/pull/673/commits/7de76b2c2eb5cec03e6f9f8d957678c972214381